mysql/README.md.jinja

```diff
@@ -61,21 +47,18 @@ db.createUser( {
 });
 ```
 
-=======
-To create a MySQL user for collectd:
+
 
 ```
- CREATE USER 'collectd'@'localhost' IDENTIFIED BY 'collectdUserPassword';
- -- Give appropriate permissions
- -- ("GRANT USAGE" is synonymous to "no privileges")
- GRANT USAGE ON *.* TO 'collectd'@'localhost';
- -- Permissions for the stats options
- GRANT REPLICATION CLIENT ON *.* TO 'collectd'@'localhost';
+db.createUser( {
+  user: "collectd",
+  pwd: "collectd",
+  roles: [ { role: "readAnyDatabase", db: "admin" }, { role: "clusterMonitor", db: "admin" } ]
+});
```

Comment:
is this to address a conflict with master? if so, what's the context for lines 53-57?

Reply:
Green task creates MySQL user for collectd, but that wasn't apparent with intro line gone (since restored)